src/CFamily/CMake/Resources.resx

```diff
@@ -117,17 +117,26 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BadCMakeSettings" xml:space="preserve">
+    <value>[CompilationDatabaseLocator] Could not parse CMakeSettings.json: {0}</value>
+  </data>
+  <data name="BadCompilationDatabaseFile" xml:space="preserve">
+    <value>[CompilationConfigProvider] Could not parse compilation database: {0}</value>
+  </data>
   <data name="FoundCompilationDatabaseFile" xml:space="preserve">
-    <value>[CompilationDatabaseLocator] Reading compilation database from '{0}</value>
+    <value>[CompilationConfigProvider] Reading compilation database from '{0}</value>
   </data>
   <data name="NoBuildConfigInCMakeSettings" xml:space="preserve">
     <value>[CompilationDatabaseLocator] Specified build configuration '{0}' could not be found at '{1}'</value>
   </data>
   <data name="NoCMakeSettings" xml:space="preserve">
-    <value>[CompilationDatabaseLocator] No {0} was found under {1}, searching for compilation database under default location: {2}</value>
+    <value>[CompilationDatabaseLocator] No {0} was found under {1}, returning default location: {2}</value>
+  </data>
+  <data name="NoCompilationDatabaseEntry" xml:space="preserve">
+    <value>[CompilationConfigProvider] Could not find file '{0}' in the compilation database.</value>
```

Comment:
File paths are often longer than the width of the output window, so putting the full file name in the middle of a message means the user has to scroll to understand the message.

If the variable part of the content can make the message harder to read, I generally move it to the end e.g.

`[CompilationConfigProvider] Could not find file in the compilation database. File: {file name}`